docs/VLCB4in4out_README.md

```diff
@@ -148,12 +158,27 @@ The following NV values define input switch function:
 
 NV Value | Function
 --------|--------
- 0 | On/Off switch
- 1 | On only push button
- 2 | Off only push button
- 3 | On/Off toggle push button
+ 0 | Do nothing
```

Comment:
Yes, this is a good change. How did this work before?

Reply:
If it was undefined then it would do nothing.  However, that wasn't documented and so I made the change.  This is important because if you now set the NV to 0 or an unspecified number, a send request is not generated and so a deleted default produced event isn't re-instated.  You need to set the NV to a defined value to allow a deleted default produced event to be re-instated.